2007-08-16  Jakub Jelinek  <jakub@redhat.com>

	* opncls.c (bfd_openr_iovec): Surround open with parentheses.

--- bfd/opncls.c.jj	2007-08-01 09:11:48.000000000 -0400
+++ bfd/opncls.c	2007-08-16 06:23:39.000000000 -0400
@@ -545,7 +545,7 @@ bfd_openr_iovec (const char *filename, c
   nbfd->filename = filename;
   nbfd->direction = read_direction;
 
-  stream = open (nbfd, open_closure);
+  stream = (open) (nbfd, open_closure);
   if (stream == NULL)
     {
       _bfd_delete_bfd (nbfd);
